guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,317 @@
+![Selection9](https://i.imgur.com/cWJB3p5.png)
+![Selection9](https://i.imgur.com/nkxml1h.png)
+
+- Go to the line that say `Uncomment the next line to enable packet forwarding for IPv4` and the remove the symbol `#` by removing the `#` sign this will make the program read and use that line to `enable packet forwarding`. 
+
+![Selection9](https://i.imgur.com/LJgDvBJ.png)
+
+- Then press `control+X` to save the config file on ubuntu and enter `Y` to confirm the file and safe the file 
+
+![Selection9](https://i.imgur.com/D2IAI1b.png)
+
+- Then press `Enter` to make sure the file name stay the same
+
+![Selection9](https://i.imgur.com/aWGEO3P.png)
+
+- Next step run command `ip link show` to see which network interfaces are up  it will look like the picture bellow
+- You should have you loopback `lo` and your network adapters `enp0s3` and `enp0s8` or similars. 
+
+![Selection9](https://i.imgur.com/2r0ghbi.png)
+
+- You will need to add another interface to the server which is going to be the ip address for your lap to go into th einternet. 
+- Run command on the terminal `sudo nano /etc/network/interfaces` and enter your admin password
+
+![Selection9](https://i.imgur.com/EmMewtr.png)
+
+- Then you will see a text file like bellow
+
+![Selection9](https://i.imgur.com/YzAyL1s.png)
+
+- Then add the new network adapter like bellow
+- ISP sever network interface 
+-	`auto enp0s8`
+		`iface enp0s8 inet static`
+			`address 172.31.11.1`
+			`netmask 255.255.255.248`
+			`up route add -net 172.0.0.0 netmask 255.0.0.0 gw 172.31.1.2`
+
+![Selection9](https://i.imgur.com/cU2izxY.png)
+
+- Then press `control X` to exit and `Y` to save file then enter to leave the file with the same name do not change the file name
+
+![Selection9](https://i.imgur.com/gCgEyWU.png)
+
+- run `sudo reboot` on the terminal this will reboot the machine and the seeting on the interfaces will take effect
+- Then run `router` on the terminal after it reboot it shout look like picture bellow 
+
+![Selection9](https://i.imgur.com/wCEIwLd.png)
+
+- Then run the next line on terminal this will add the new network adapter to your network route. 
+- `sudo route add –net 172.0.0.0 netmask 255.0.0.0 gw 172.31.1.3 dev enp0s8`
+
+![Selection9](https://i.imgur.com/AveqQd6.png)
+
+- Then run `route` again to verify it took effect 
+
+![Selection9](https://i.imgur.com/I9QDTpD.png)
+
+- Then run `sudo services networking restart` to restard the network services nothing will display it only going to ask you for your admin password. This will restard your network services for the code above take effect. 
+
+![Selection9](https://i.imgur.com/hR7cdtQ.png)
+
+- Then to verify everything is working fine run `ifconfig` the enp0s8 should have the infomation you enter like bellow
+
+![Selection9](https://i.imgur.com/b4lVh9W.png)
+
+- Before running the following command run `sudo apt-get update and upgrade` and `sudo apt-get install iptables-persistant` this will make the followin lines work correctly.  
```

Comment:
Change:
Before running the following command run `sudo apt-get update and upgrade` and `sudo apt-get install iptables-persistant` this will make the followin lines work correctly.  
To:
Run  `sudo apt-get install iptables-persistant`, if this results in an error then run `sudo apt-get update && sudo apt-get upgrade -y`.